meson.build

```diff
@@ -0,0 +1,101 @@
+project(
+    'Kromblast',
+    'cpp',
+    version : '0.1',
+    default_options : ['warning_level=3']
+)
+
+cc = meson.get_compiler('cpp')
+
+# Extra Dependencies
+
+webview_depends = [
+    dependency('gtk+-3.0'),
+    dependency('webkit2gtk-4.0')
+]
+
+webview_lib = library(
+    'webview',
+    'src/libs/webview/webview.h',
```

Comment:
Does this have any source code too? :eyes: (Header files do not get compiled.)

Reply:
Can i have more information (the error when you tried to build). Also the code of the webview is not from me ([the original repository](https://github.com/webview/webview)). And are you on windows (i don't tried on windows so it maybe doesn't work).